testutils/zone.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2015 The Cockroach Authors.
```

Comment:
What would be the benefit? It's certainly not implied by this change, which was only made so that the code can be shared between unit tests and acceptance tests (acceptance tests don't want to link in stdlib's "testing").